doc/build_docs.py

```diff
@@ -38,6 +38,34 @@
 _DOCTREE_TMP = _TMP / "doctree"
 
 
+class ShardingStrategy:
+    def __init__(self, spec: str) -> None:
+        self.spec = spec
+        if spec == "no":
```

Comment:
maybe docstring or comments summarising the three modes?

Reply:
> maybe docstring or comments summarising the three modes?

Done.